Sources/Player/Common/Logging/PlayerLoggable.swift

```diff
@@ -2,13 +2,17 @@ import Common
 import Foundation
 import Logging
 
+// MARK: - Constants
```

Comment:
weird, why my formatter formatted some things and not others 🤔 maybe I'll do the setup again just in case

Reply:
This is my setup
```
➜  ~ swiftlint --version
0.55.1
```